pkg/volume/volume.go

```diff
@@ -103,7 +103,7 @@ func Describe(vols []string, openebsNs string) error {
 		if openebsNs == "" {
 			if val, ok := nsMap[casType]; ok {
 				k.Ns = val
-			} else if casType != util.ZFSCasType && casType != util.LVMCasType {
+			} else if casType != util.ZFSCasType && casType != util.LVMCasType && casType != util.LocalHostpath {
```

Comment:
I had almost forgot why I had this code-quirk, serves me right for incomplete code-comments🤣 

IIRC, this is required because the CSI provisioners and _now the external-provisioners_ of these volumes are present in `kube-system` namespace and thus if `GetOpenEBSNamespaceMap()` had to figure out `cas-type→namespace relation`, it'd likely figure it out incorrectly as the namespace will not be `kube-system` i.e. the relevant custom resources if any would be present on some other namespace where the provisioner is actually present.


As of now, my memory is a bit hazy and the error message and the core around it seems a bit off, I'll re-read and confirm if this code change is required.

Reply:
@Abhishek-kumar09 This change isn't required, as the namespace of LocalHostPath resource can be guessed correctly via the `nsMap` variable